clownOverrides/.npmignore

```diff
@@ -0,0 +1 @@
+!clown/**/src
```

Comment:
This was unnecessary, you could've just added it to the original `.nmignore` without making an override. I know it's confusing!

Reply:
Ummm, no that's not guaranteed. If `src` gets removed from `.npmignore`, clown will add it after `src`. But you're right, the way you have it guarantees that it will be added after `src` 😄

Reply:
I guess we should revert e798ca3d33a1fe75ab85f173d3cdc7172c600264 then